plugins/node.d/processes

```diff
@@ -130,7 +130,7 @@ if [ "$1" = "config" ]; then
 
     # OS specific flags
     if [ "$OPERSYS" = "Linux" ]; then
-        echo "graph_order sleeping stopped zombie dead paging uninterruptible runnable processes"
+        echo "graph_order sleeping idle stopped zombie dead paging uninterruptible runnable processes"
```

Comment:
I think, this would change the color of the fields in the graph (please check - maybe I am wrong).  In this case it would be great, if you moved it to the end of the list.

Reply:
I don't know why it should, as the colors are defined via
```
# Define colours
RUNNABLE='22ff22'         # Green
SLEEPING='0022ff'         # Blue
STOPPED='cc0000'          # Darker red
ZOMBIE='990000'           # Darkest red
UNINTERRUPTIBLE='ffa500'  # Orange
IDLE='4169e1'             # Royal blue
PAGING='00aaaa'           # Darker turquoise
INTERRUPT='ff00ff'        # Fuchsia
LOCK='ff3333'             # Lighter red
RUNNING='00ff7f'          # Spring green
DEAD='ff0000'             # Red
SUSPENDED='ff1493'        # Deep pink
TOTAL='c0c0c0'            # Silver
```

The definition matches the resulting [color table](https://user-images.githubusercontent.com/7456652/52110155-0c50fb00-2600-11e9-82b5-270c7953a1f3.png).